app/assets/stylesheets/blocks/_flash.scss

```diff
@@ -0,0 +1,5 @@
+.callout {
+  &.notice {
+    @include callout-style(map-get($foundation-palette, success));
+  }
+}
```

Comment:
In Fincap, we have Foundation stuff like this under `app/assets/stylesheets/foundation`. I'd be tempted to do something similar here, to make clear things are completely custom styles and which are doing Foundation mods. Something to ponder

Reply:
Quite agree on the idea. I don't have time for checking what's overriding Foundation styles though :/

Reply:
Fair enough! It's too much to unravel right now. 

I recognised `.callout`, `.tabs-content`, and `.top-bar` as Foundation classes.